.cargo/config.toml

```diff
@@ -1,3 +1,7 @@
+[build]
+rustflags = ["-Ctarget-cpu=native"]
```

Comment:
Will pgvecto.ra be distributed as binary or only source code, if user install this as extension?

Reply:
Source code, I think.